mpas_analysis/ocean/aste_transects.py

```diff
@@ -0,0 +1,431 @@
+# This software is open source software available under the BSD-3 license.
+#
+# Copyright (c) 2022 Triad National Security, LLC. All rights reserved.
+# Copyright (c) 2022 Lawrence Livermore National Security, LLC. All rights
+# reserved.
+# Copyright (c) 2022 UT-Battelle, LLC. All rights reserved.
+#
+# Additional copyright and license information can be found in the LICENSE file
+# distributed with this code, or at
+# https://raw.githubusercontent.com/MPAS-Dev/MPAS-Analysis/master/LICENSE
+import xarray as xr
+import numpy
+import os
+
+from collections import OrderedDict
+
+from mpas_analysis.shared import AnalysisTask
+from mpas_analysis.ocean.compute_transects_subtask import \
+    TransectsObservations
+
+from mpas_analysis.ocean.compute_transects_with_vel_mag import \
+    ComputeTransectsWithVelMag
+
+from mpas_analysis.ocean.plot_transect_subtask import PlotTransectSubtask
+
+from mpas_analysis.shared.io.utility import build_config_full_path, \
+    make_directories, build_obs_path
+from mpas_analysis.shared.io import write_netcdf
+
+
+class AsteTransects(AnalysisTask):
+    """
+    Plot model output at Arctic transects and compare it against ASTE
+    reanalysis data
+    """
+
+    # Authors
+    # -------
+    # Milena Veneziani, Kat Smith, Alice Barthel, Xylar Asay-Davis
+
+    def __init__(self, config, mpasClimatologyTask, controlConfig=None):
+        """
+        Construct the analysis task and adds it as a subtask of the
+        ``parentTask``.
+
+        Parameters
+        ----------
+        config : mpas_tools.config.MpasConfigParser
+            Configuration options
+
+        mpasClimatologyTask : ``MpasClimatologyTask``
+            The task that produced the climatology to be remapped and plotted
+            as a transect
+
+        controlconfig : mpas_tools.config.MpasConfigParser, optional
+            Configuration options for a control run (if any)
+        """
+        # Authors
+        # -------
+        # Milena Veneziani, Kat Smith, Alice Barthel, Xylar Asay-Davis
+
+        tags = ['climatology', 'transect', 'aste', 'publicObs', 'arctic']
+
+        # call the constructor from the base class (AnalysisTask)
+        super(SoseTransects, self).__init__(
```

Comment:
Should this be AsteTransects?
```suggestion
        super(AsteTransects, self).__init__(
```

Reply:
yes, there are many places where I haven't completed my changes, mostly because I want to wait for when the ASTE processed data is ready and the important changes to the script can be put in.